apps/general/models.py

```diff
@@ -1,3 +1,230 @@
+    gstn_password = models.CharField(max_length=255, default=None, null=True) 
+    other_license_1 = models.CharField(max_length=255, default=None, null=True)
+    other_license_2 = models.CharField(max_length=255, default=None, null=True)
+    picture = models.ImageField(max_length=255, default=None, null=True, upload_to=branches_picture) 
+    address = models.TextField(default=None, null=True)
+    country = models.CharField(max_length=50, default=None, null=True)
+    state = models.CharField(max_length=50, default=None, null=True)
+    city = models.CharField(max_length=50, default=None, null=True)
+    pin_code = models.CharField(max_length=20, default=None, null=True)
+    phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number must be entered in the format: '+999999999'. Up to 15 digits allowed.")
+    phone = models.CharField(validators=[phone_regex], max_length=20, default=None, null=True)  # validators should be a list
+    email = models.EmailField(max_length=255, default=None, null=True)
+    longitude = models.DecimalField(max_digits=10, decimal_places=7, default=None, null=True)
+    latitude = models.DecimalField(max_digits=10, decimal_places=7, default=None, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f"{self.branch_id}.{self.name}"
+    
+    class Meta:
+        db_table = 'branches' 
+
+    def save(self, *args, **kwargs):
+        if self.e_way_password:
+            # Hash the e_way_password using bcrypt and save as bytes
+            hashed_eway_password = bcrypt.hashpw(self.e_way_password.encode(), bcrypt.gensalt())
+            self.e_way_password = hashed_eway_password
+        if self.gstn_password:
+            # Hash the gstn_password using bcrypt and save as bytes
+            hashed_gstn_password = bcrypt.hashpw(self.gstn_password.encode(), bcrypt.gensalt())
+            self.gstn_password = hashed_gstn_password
+        super().save(*args, **kwargs)
+
+    def verify_e_way_password(self, password):
+        # Verify e_way_password using bcrypt
+        if self.e_way_password:
+            return bcrypt.checkpw(password.encode(), self.e_way_password)
+        return False
+
+    def verify_gstn_password(self, password):
+        # Verify gstn_password using bcrypt
+        if self.gstn_password:
+            return bcrypt.checkpw(password.encode(), self.gstn_password)
+        return False
+    
+
+# Dummy encryption and decryption functions for demonstration purposes
+def encrypt(text):
+    # Encode the text using base64
+    encoded_bytes = base64.b64encode(text.encode("utf-8"))
+    encrypted_text = encoded_bytes.decode("utf-8")
+    return encrypted_text
+
+def decrypt(encrypted_text):
+    # Decode the text using base64
+    decoded_bytes = base64.b64decode(encrypted_text.encode("utf-8"))
+    decrypted_text = decoded_bytes.decode("utf-8")
+    return decrypted_text
```

Comment:
Remove above dummy methods if thery are not needed.

Reply:
Yeah it is needed Thulasi for encrypting and decrypting the account number .I mentioned it dummy as we can customize this code Thulasi.